pointers/pointers.c

```diff
@@ -21,6 +26,17 @@ void string_copy(char *x, char *y)
 */
 char *find_char(char *str, int c)
 {
+    char *n;
+    int i;
+
+    for( i = 0; i != '\0'; i++) {
```

Comment:
you're not referencing the string here, just comparing the increment to the null character.